pandas/tseries/tests/test_resample.py

```diff
@@ -2839,6 +2839,32 @@ def test_aggregate_with_nat(self):
             # if NaT is included, 'var', 'std', 'mean', 'first','last'
             # and 'nth' doesn't work yet
 
+    def test_resample_with_nat(self):
```

Comment:
if you move these to ``Base`` and use ``.create_series()`` you will get testing for free for all of the sub-classes (DTI, PeriodIndex, TimedeltaIndex)